### PR TITLE
Extract kernel, bias values of a layer

### PR DESCRIPTION
**Pull request by shravankumar147**
This function will extract the kernel, and bias values of given tensor-flow model graph layer.

### CODE REVIEW

**Comment by shravankumar147** [comment]
`I signed it!`

**Review by kaushik1094 — COMMENTED** [review]
good job

**Comment by protoget** [comment]
Re @kaushik1094 does the PR look good to you?

**Comment by kaushik1094** [comment]
@protoget Yes, It looks good for me.

**Review by protoget — APPROVED** [review]
Grant LGTM on behalf of kaushik1094@

File: tensorflow/python/util/get_variable_values.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    P_Wb = {}
+
+    for i in tf.get_collection(tf.GraphKeys.TRAINABLE_VARIABLES, scope=layer):
+        m = re.search(r'\w+\/\w+\:\d', str(i)).group(0)
+        M.append(m)
+        v = m.split(":")[0]
+        param_type = ''.join(v.split("/")[1])
+        P.append(param_type)
+        print(("saving: {}".format(m)))
+    param_dict[P[0]] = graph.get_tensor_by_name(M[0]).eval()  # Weights
```
Comment (protoget) [inline tensorflow/python/util/get_variable_values.py]: I dont' think this would work in a non-interactive session.

**Review by protoget — CHANGES_REQUESTED** [review]
Undo LSTM

File: tensorflow/python/util/get_variable_values.py (hunk 1 above)
Comment (shravankumar147) [reply tensorflow/python/util/get_variable_values.py]: How can it be used in non-interactive session. Initially I thought of this to look into parameters while addressing vanishing/exploding gradients problem.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @protoget: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @protoget: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @protoget: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @protoget: It has been 86 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @protoget: It has been 101 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @protoget: It has been 116 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Pull request closed without merging**